.github/workflows/ci-actions.yml

```diff
@@ -636,7 +636,7 @@ jobs:
           # done because there is no good way to pass strings with empty values to the previous command
           # so this hack is as good as any
           if [ "$CATEGORY" == "Misc1" ]; then
-            mvn -Dnative -Dquarkus.native.container-build=true -B --settings .github/mvn-settings.xml -f 'integration-tests/simple with space/' verify
+            mvn -Dnative -Dquarkus.native.container-build=true -Dquarkus.native.builder-image=quay.io/quarkus/ubi-quarkus-native-image:20.1.0-java11 -B --settings .github/mvn-settings.xml -f 'integration-tests/simple with space/' verify
```

Comment:
This is already the default value. See https://github.com/quarkusio/quarkus/blob/edd539f29c305ab908ed6bcd315821e1e5beb137/core/deployment/src/main/java/io/quarkus/deployment/pkg/NativeConfig.java#L143

Reply:
:+1: didn't know that.
In that case I think https://github.com/quarkusio/quarkus/blob/master/.github/workflows/ci-actions.yml#L37 should change to contain only `-Dquarkus.native.container-build=true` for consistency and to avoid similar mistakes to the one fixed by #10178.

Reply:
I agree, not sure why it was explicitly set

Reply:
Shall I open a new PR or force push here and edit the title?

Reply:
Up to you 😁, I guess force pushing and editing the title is fine, I guess